modules/kmm-deploying-modules.adoc

```diff
@@ -0,0 +1,197 @@
+cluster, through a ModuleLoader image.
+A `Module` CRD specifies one or more kernel versions with which it is compatible, and a node selector.
+
+The compatible versions for a `Module` are listed under `.spec.moduleLoader.container.kernelMappings`.
+A kernel mapping can either match a `literal` version, or use `regexp` to match many of them at the same time.
+
+The reconciliation loop for `Module` runs the following steps:
+
+. List all nodes matching `.spec.selector`.
+. Build a set of all kernel versions running on those nodes.
+. For each kernel version, go through `.spec.moduleLoader.container.kernelMappings` and find the appropriate container image name.
+
+If the kernel mapping has `build` or `sign` defined and the container image does not already exist, run the build and / or the signing job as needed.
+
+. Create a ModuleLoader `DaemonSet` with the container image determined at the previous step.
+. If `.spec.devicePlugin` is defined, create a device plugin `DaemonSet` using the configuration specified under `.spec.devicePlugin.container`.
+. Run garbage-collect on:
+ .. Existing `DaemonSets` targeting kernel versions that are not run by any node in the cluster.
+ .. Successful build jobs.
+ .. Successful signing jobs.
+
+== Security and permissions
+
+[NOTE]
+====
+Loading kernel modules is a highly sensitive operation.
+Once loaded, kernel modules have all possible permissions to do any kind of operation on the node.
+====
+
+=== `ServiceAccounts` and `SecurityContextConstraints`
+
+KMM creates a privileged workload to load the kernel modules on nodes.
+That workload needs `ServiceAccounts` enabled to use the `privileged` `SecurityContextConstraint` (SCC).
+
+The authorization model for that workload depends on the `Module` namespace, as well as its spec.
+
+* If the `.spec.moduleLoader.serviceAccountName` or `.spec.devicePlugin.serviceAccountName` fields are set, they are always used.
+* If those fields are not set, then.
+ ** If the `Module` is created in the operator's namespace (`openshift-kmm` by default), then KMM will use its default, powerful `ServiceAccounts` to run the DaemonSets.
+ ** If the `Module` is created in any other namespace, then KMM will run the DaemonSets as the namespace's `default` `ServiceAccount`. The `Module` cannot run privileged workload unless you manually enable it to use the `privileged` SCC.
+
+[NOTE]
+====
+`openshift-kmm` is a trusted namespace.
+
+When setting up RBAC permissions, keep in mind that any user or ServiceAccount creating a `Module` resource in the `openshift-kmm` namespace will result in KMM automatically running privileged workloads on potentially all nodes in the cluster.
+====
+
+To allow any `ServiceAccount` to use the `privileged` SCC and hence to run ModuleLoader and / or device plugin pods, use the following command.
+
+[,shell]
+----
+oc adm policy add-scc-to-user privileged -z "${serviceAccountName}" [ -n "${namespace}" ]
+----
+
+=== Pod Security standards
```

Comment:
```suggestion
=== Pod Security Standards
```
Kubernetes docs [do capitalize the S](https://kubernetes.io/docs/concepts/security/pod-security-standards/), however I will leave that to your appreciation.